sky/backends/onprem_utils.py

```diff
@@ -153,7 +154,7 @@ def get_local_cluster_config_or_error(cluster_name: str) -> Dict[str, Any]:
     raise ValueError(f'Cluster config {local_file} not found.')
 
 
-def check_local_installation(ips: List[str], auth_config: Dict[str, str]):
+def check_and_install_local_env(ips: List[str], auth_config: Dict[str, str]):
     """Checks if the Sky dependencies are properly installed on the machine.
```

Comment:
* `Checks if SkyPilot dependencies are present on the machine. Installs them if not already installed.` or something along these lines

Reply:
Done, thanks!